collections/ansible_collections/cloudkit/service/roles/extract_vm_template_info/tasks/main.yaml

```diff
@@ -0,0 +1,17 @@
+- name: Extract template id
+  when: >-
+    template_id is not defined
+  ansible.builtin.set_fact:
+    template_id: "{{ vm_order.spec.templateID }}"
+
+- name: Handle unqualfied template ids
+  when: >-
+    "." not in template_id
+  ansible.builtin.set_fact:
+    template_id: "cloudkit.templates.{{ template_id }}"
```

Comment:
not sure we should handle that, from what I remember the templates are published with their fqdn to the fulfillment-service.